api/config/authz/config_request.go

```diff
@@ -47,8 +47,13 @@ func (c *ConfigRequest) Validate() error {
 	projectLimit := c.GetV1().GetSys().GetService().GetProjectLimit()
 
 	if projectLimit != nil {
-		if limit := projectLimit.GetValue(); limit < constants_v2.MaxProjects {
-			e := errors.Errorf("project limit must be at least %v", constants_v2.MaxProjects)
+		if limit := projectLimit.GetValue(); limit < constants_v2.MinConfigurableProjects {
```

Comment:
While I agree with this change to a lower limit check, not clear that it should ever be removed--the system needs to allow for at least 1 project I would think. So consider setting it to 1 now and be done with it...?

Reply:
@msorens We could, but once we do away with the limit, we won't allow customers to configure their limit at all, and this will all go away.